ha/util/message_bus.py

```diff
@@ -106,7 +107,12 @@ def run(self):
             try:
                 if not retry:
                     message = self.consumer.receive(timeout=0)
-                status = self.callback(message)
+                try:
+                    status = self.callback(message)
+                except Exception as e:
```

Comment:
same variable 'e' is used to catch exception here and in the outer try catch.
Can you please confirm the scope of 'e' is fine wrt both inner and outer try except 's

Reply:
yes both have different scope 
Exception as e
so it will define by exception

also its scope will be closed as soon as exception block end